http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -69,9 +69,20 @@ class GceInterpreter[F[_]: Parallel: ContextShift](
         SetUpProjectFirewallsParams(params.runtimeProjectAndName.googleProject, network)
       )
 
+      zoneParam <- params.runtimeConfig match {
```

Comment:
Changes in this file LGTM, but think we need equivalent in `DataprocInterpreter`.

Or, if we want to punt Dataproc regionality for this PR, remove it from the API/swagger page.